docs/guides/capsule-update.md

```diff
@@ -108,8 +119,13 @@ should have this feature enabled.
 
     Don't reboot or power off the device until the process is completed!
 
-After a successful update, you should have an updated firmware with default
-settings.
+After either a successful or failed update, the machine should reboot
+ automatically. After that if everything succeeded you should have an
+ updated firmware with migrated data from the previous version.
```

Comment:
I think the words should be swapped:
```suggestion
 updated firmware with data migrated from the previous version.
```

Reply:
https://github.com/Dasharo/docs/pull/912/commits/80dcc0ef34936f01a8f08d51fbba747f9f6656d6